### PR TITLE
Handle an async design system

### PR DESCRIPTION
**Pull request by thecrypticace**
In v4 we've made compile async as well as `__unstable_loadDesignSystem` and we need to handle that in intellisense.

### CODE REVIEW

File: packages/tailwindcss-language-server/src/util/v4/design-system.ts [hunk 1]
```diff
@@ -37,7 +37,11 @@ export async function loadDesignSystem(
   let resolved = await resolveCssImports().process(css, { from: filepath })
 
   // Step 3: Take the resolved CSS and pass it to v4's `loadDesignSystem`
-  let design = tailwindcss.__unstable__loadDesignSystem(resolved.css) as DesignSystem
+  let design: DesignSystem = await tailwindcss.__unstable__loadDesignSystem(resolved.css, {
+    loadPlugin() {
+      return () => {}
```
Comment (thecrypticace) [inline packages/tailwindcss-language-server/src/util/v4/design-system.ts:42]: We're intentionally not handling `@plugin` here — this PR is purely to make sure alpha 19 doesn't break in intellisense. That support will be done in a separate PR.

**Review by RobinMalfait — APPROVED** [review]
